utils/hct/HashStability.py

```diff
@@ -0,0 +1,142 @@
+
```

Comment:
Can we introduce some tests for this script?

Reply:
Added one test as FileCheck test.